README.md

```diff
@@ -52,22 +59,54 @@ To start the Clearwater services, run:
 
 #### Scaling the deployment
 
-Having started up a deployment, it is then possible to scale it by adding more Sprout, Memcached, Chronos or Cassandra nodes.   E.g. to spin up an additional node of each of these types, run:
+Having started up a deployment, it is then possible to scale it by adding more Sprout, Astaire, Chronos or Cassandra nodes.   E.g. to spin up an additional node of each of these types, run:
 
-    sudo docker-compose -f minimal-distributed.yaml scale sprout=2 memcached=2 chronos=2 cassandra=2
+    sudo docker-compose -f minimal-distributed.yaml scale sprout=2 astaire=2 chronos=2 cassandra=2
 
 Note that scaling of Docker deployments is a work in progress and there are currently a number of known issues...
 
 * Bono doesn’t automatically start using new Sprout nodes unless the Bono process is restarted, e.g.
 
     `sudo docker exec clearwaterdocker_bono_1 sudo service bono restart`
-    
+
 * Homestead-prov and Ellis don’t load balance across multiple Cassandra nodes.
 
 * There is no tested or documented process for scaling down clusters of storage nodes in Docker -- it isn't sufficient to just delete the containers as they need to be explicitly decommissioned and removed from the clusters.
 
 If you scale up the clusters of storage nodes, you can monitor progress as new nodes join the clusters by running `utils/show_cluster_state.sh`.
 
+## Using Kubernetes
+
+Instead of using Docker Compose, you can deploy Clearwater in Kubernetes. This
+requires a Kubernetes cluster, and a Docker repository.
+
+### Prepare the images and edit the yaml files
+
+- First, build all the required images locally.
+
+        # Build the Clearwater docker images.
+        cd clearwater-docker
+        for i in base astaire cassandra chronos bono ellis homer homestead ralf sprout ; do docker build -t clearwater/$i $i ; done
```

Comment:
This error disappeared somehow ...  Now the only problem I can see is homestead:
```
20-04-2017 13:27:32.102 UTC Status load_monitor.cpp:105: Constructing LoadMonitor
20-04-2017 13:27:32.102 UTC Status load_monitor.cpp:106:    Target latency (usecs)   : 100000
20-04-2017 13:27:32.102 UTC Status load_monitor.cpp:107:    Max bucket size          : 1000
20-04-2017 13:27:32.102 UTC Status load_monitor.cpp:108:    Initial token fill rate/s: 100.000000
20-04-2017 13:27:32.102 UTC Status load_monitor.cpp:109:    Min token fill rate/s    : 10.000000
20-04-2017 13:27:32.102 UTC Status dnscachedresolver.cpp:150: Creating Cached Resolver using servers:
20-04-2017 13:27:32.102 UTC Status dnscachedresolver.cpp:160:     10.96.0.10
20-04-2017 13:27:32.102 UTC Status a_record_resolver.cpp:54: Created ARecordResolver
20-04-2017 13:27:32.102 UTC Status a_record_resolver.cpp:54: Created ARecordResolver
20-04-2017 13:27:32.102 UTC Status cassandra_store.cpp:181: Configuring store connection
20-04-2017 13:27:32.102 UTC Status cassandra_store.cpp:182:   Hostname:  cassandra.default.svc.cluster.local
20-04-2017 13:27:32.103 UTC Status cassandra_store.cpp:183:   Port:      9160
20-04-2017 13:27:32.103 UTC Status cassandra_store.cpp:211: Configuring store worker pool
20-04-2017 13:27:32.103 UTC Status cassandra_store.cpp:212:   Threads:   10
20-04-2017 13:27:32.103 UTC Status cassandra_store.cpp:213:   Max Queue: 0
20-04-2017 13:27:32.114 UTC Error main.cpp:756: Failed to initialize the Cassandra cache with error code 1.
20-04-2017 13:27:32.114 UTC Status main.cpp:757: Homestead is shutting down
```

Not sure about the reason.